python/semantic_kernel/contents/chat_message_content.py

```diff
@@ -27,9 +29,10 @@
     TEXT_CONTENT_TAG: TextContent,
     FUNCTION_CALL_CONTENT_TAG: FunctionCallContent,
     FUNCTION_RESULT_CONTENT_TAG: FunctionResultContent,
+    IMAGE_CONTENT_TAG: ImageContent,
 }
 
-ITEM_TYPES = Union[TextContent, StreamingTextContent, FunctionResultContent, FunctionCallContent]
+ITEM_TYPES = Union[TextContent, StreamingTextContent, FunctionResultContent, FunctionCallContent, ImageContent]
```

Comment:
Does this need a `Union` now? Can't we use the pipe `|`?

Reply:
yes, this is a typevar and not a annotation!